api/controllers/budgetControllers.js

```diff
@@ -0,0 +1,39 @@
+const Budget = require('../models/budget');
+
+const budgetCreate = (req, res) => {
+    const { title, budgetAmount } = req.body;
+    const newBudget = new Budget({ title, budgetAmount });
+    newBudget.save(newBudget, (err, savedBudget) => {
+        if(err) { res.status(500).json(err); return};
+        res.json(savedBudget);
+    });
+};
+
+// const budgetGetAll = (req, res) => {
+//     Budget.find({})
+//     .select('title')
+//     .exec()
+//     .then(budgets => {
+//         if (budget.length === 0) throw new Error();
+//         res.json(budgets);
+//     })
+//     .catch(err => res.status(422).json(err));
+// };
+
+// const budgetGetById = (req, res) => {
+//     const { id } = req.params;
+//     Budget.findById(id)
+//     .populate('title budgetAmount') //not sure about this line
```

Comment:
You would actually want to select two fields and the data you want them to be populated with here. Right now you've only selected the fields. You haven't chosen what data needs to be populated. But you're on the right track for sure! 

Also, cleanest code award! Thinks are tight and clean!

Reply:
TY! I try to make it halfway presentable lol

Reply:
yeah I wasn't sure how to do that.  I think that's where that syntax below came into play lol.